source/agora/test/Base.d

```diff
@@ -74,7 +75,15 @@ private UnitTestResult customModuleUnitTester ()
     auto filter = environment.get("dtest").toLower();
     size_t filtered;
 
-    UnitTestResult result;
+    // can't use ModuleInfo[], opApply returns temporaries..
+    struct ModTest
+    {
+        string name;
+        void function() test;
+    }
+
+    ModTest[] mod_tests;
+
     foreach (ModuleInfo* mod; ModuleInfo)
```

Comment:
So, interesting thing about this. You can't store this pointer (or its value), if you try to store it and use it outside the loop, it crashes. Strange implementation of this feature..